pkg/cmd/server/api/latest/helpers.go

```diff
@@ -76,17 +76,25 @@ func WriteYAML(obj runtime.Object) ([]byte, error) {
 	return content, err
 }
 
+func ReadYAML(data []byte, obj runtime.Object) error {
```

Comment:
Add a TODO.  This should go away once we have the yaml serializer in the rebase after this one.